ext/sockets/tests/socket_cmsg_credentials_fbsd.phpt

```diff
@@ -0,0 +1,92 @@
+--TEST--
+recvmsg(): receive SCM_CREDS messages
+--EXTENSIONS--
+sockets
+--SKIPIF--
+<?php
+
+if (!str_contains(PHP_OS, 'FreeBSD')) {
+    die('supported only on FreeBSD');
+}
+--CLEAN--
+<?php
+$path = __DIR__ . "/socket_cmsg_credentials_fbsd.sock";
+@unlink($path);
+--FILE--
+<?php
+include __DIR__."/mcast_helpers.php.inc";
+$path = __DIR__ . "/socket_cmsg_credentials_fbsd.sock";
+
+@unlink($path);
+
+echo "creating send socket\n";
+$sends1 = socket_create(AF_UNIX, SOCK_DGRAM, 0) or die("err");
+var_dump($sends1);
+socket_set_nonblock($sends1) or die("Could not put in non-blocking mode");
+
+echo "creating receive socket\n";
+$s = socket_create(AF_UNIX, SOCK_DGRAM, 0) or die("err");
+var_dump($s);
+$br = socket_bind($s, $path) or die("err");
+var_dump($br);
+socket_set_nonblock($sends1) or die("Could not put in non-blocking mode");
```

Comment:
```suggestion
socket_set_nonblock($s) or die("Could not put in non-blocking mode");
```

Reply:
ah right I copied from the original test, did not see it :-)